cmd/pool/account.go

```diff
@@ -247,6 +250,106 @@ func listAccounts(ctx *cli.Context) error {
 	return nil
 }
 
+var updateExpiryCommand = cli.Command{
+	Name:      "expire",
```

Comment:
`expire` sounds like it will close the account. Should all it `renew` instead?

Reply:
+1 for `renew`